web/README.md

```diff
@@ -1,31 +1,42 @@
-# Static Site
+# Foreman documentation site
 
-Landing page for [docs.theforeman.org](https://docs.theforeman.org) is available as a [Hugo](https://gohugo.io/) project.
+Uses https://nanoc.app/
 
-## Preparing the environment
+Synopsis:
 
-Install `hugo`, for example on a Fedora system:
+	# bundle config set --local path 'vendor/bundle'
```

Comment:
`#` is the convention for root. This does not look like it should run as root. Repeated comment in this file.